Jenkinsfile

```diff
@@ -7,11 +12,17 @@ def buildApps(appList) {
   }
 }
 
-def runningOn(machine) {
-  println "Stage running on:"
-  println machine
+def buildDocs(){
+  sh 'pip install git+ssh://git@github.com/xmos/xmosdoc'
+  sh 'xmosdoc html latex'
```

Comment:
This is now 'pdf' and not 'latex' although it will still work for compatibility.
I'd also recommend adding linkcheck back in or just leaving it as the default which is `linkcheck html pdf`.